docs/integration_tests.txt

```diff
@@ -4,5 +4,5 @@
 
 cargo nextest list
 
-cargo nextest run {TESTNAME} --nocapture
+cargo nextest run --features test {TESTNAME} --nocapture
```

Comment:
add deprecations to features list?
also i think --nocapture will clutter the console as a default?

Reply:
It was here beforehand, so beyond the scope of this PR...and I agree it clutters things up when running all tests, it's quite nice for running individual tests, though.